src/index.js

```
@@ -72,121 +60,121 @@ fastify.get('/health', {
+          required: ['method'],
+          description: 'Verification configuration',
+          properties: {
+            method: { type: 'string', description: 'Verification method (e.g. online)' },
+            verifierName: { type: 'string', description: 'Name of the verifier (e.g. dhiway)' }
+          },
+          allOf: [
+            {
+              if: {
+                properties: { method: { const: 'online' } }
+              },
+              then: {
```

Biome (1.9.4)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)